src/plugins/gui/fiatProviderTypes.ts

```diff
@@ -25,13 +25,24 @@ export interface FiatProviderQuote {
   closeQuote: () => Promise<void>
 }
 
-export type FiatProviderQuoteErrorTypes = 'overLimit' | 'underLimit' | 'regionRestricted' | 'assetUnsupported' | 'paymentUnsupported'
+export type FiatProviderQuoteErrorTypes =
+  | 'overLimit'
+  | 'overUnknownLimit'
```

Comment:
This is overkill to add new error types. Just make the `errorAmount` optional and have the plugin show a different error if the amount isn't provided.

Reply:
The intent was to avoid special casing